src/stdbuf/libstdbuf.rs

```diff
@@ -47,12 +42,12 @@ fn set_buffer(stream: *mut FILE, value: &str) {
 #[no_mangle]
 pub extern fn stdbuf() {
     if let Ok(val) = env::var("_STDBUF_E") {
-        set_buffer(stderr, &val);
+        set_buffer(STDERR_FILENO as *mut FILE, &val);
```

Comment:
No problem. :)

`stdbuf` likely broke when we updated its command-line processing, as I'm pretty sure it worked before.